src/olympia/editors/forms.py

```diff
@@ -144,18 +122,8 @@ def clean_addon_type_ids(self):
         return self.cleaned_data['addon_type_ids']
 
     def clean_application_id(self):
```

Comment:
afaik we can just leave out the function if we're not doing any special cleaning and django forms does this anyway.